### ISSUE TITLE
Open Refine 2.8 exception at start on windows 10 

### ISSUE THREAD

**Issue by srht** [OpenRefine/OpenRefine#1441]
There is an exception at start when trying to open OpenRefine 2.8 version:

```
14:34:04.611 [            refine_server] Starting Server bound to '127.0.0.1:3333' (0ms)
14:34:04.622 [            refine_server] Initializing context: '/' from 'C:\openrefine\webapp' (11ms)
14:34:05.031 [                   refine] Starting OpenRefine 2.8 [TRUNK]... (409ms)
14:34:05.036 [       FileProjectManager] Failed to load workspace from any attempted alternatives. (5ms)
Exception in thread "main" java.lang.ExceptionInInitializerError
        at org.python.util.PythonInterpreter.<init>(PythonInterpreter.java:100)
        at org.python.util.PythonInterpreter.<init>(PythonInterpreter.java:94)
        at org.python.util.PythonInterpreter.<init>(PythonInterpreter.java:71)
        at com.google.refine.jython.JythonEvaluable.<clinit>(JythonEvaluable.java:83)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:161)
        at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:247)
        at org.mozilla.javascript.optimizer.OptRuntime.callProp0(OptRuntime.java:119)
        at org.mozilla.javascript.gen.c4._c1(file:/C:/openrefine/webapp/extensions/jython/module/MOD-INF/controller.js:46)
        at org.mozilla.javascript.gen.c4.call(file:/C:/openrefine/webapp/extensions/jython/module/MOD-INF/controller.js)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:398)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3065)
        at org.mozilla.javascript.gen.c4.call(file:/C:/openrefine/webapp/extensions/jython/module/MOD-INF/controller.js)
        at edu.mit.simile.butterfly.ButterflyModuleImpl.scriptInit(ButterflyModuleImpl.java:636)
        at edu.mit.simile.butterfly.ButterflyModuleImpl.init(ButterflyModuleImpl.java:94)
        at edu.mit.simile.butterfly.Butterfly.initializeModule(Butterfly.java:476)
        at edu.mit.simile.butterfly.Butterfly.configure(Butterfly.java:451)
        at edu.mit.simile.butterfly.Butterfly.init(Butterfly.java:308)
        at org.mortbay.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:440)
        at org.mortbay.jetty.servlet.ServletHolder.doStart(ServletHolder.java:263)
        at com.google.refine.RefineServer.configure(Refine.java:296)
        at com.google.refine.RefineServer.init(Refine.java:208)
        at com.google.refine.Refine.init(Refine.java:114)
        at com.google.refine.Refine.main(Refine.java:108)
Caused by: java.lang.IllegalArgumentException: Cannot create PyString with non-byte value
        at org.python.core.PyString.<init>(PyString.java:57)
        at org.python.core.PyString.<init>(PyString.java:70)
        at org.python.core.PyString.<init>(PyString.java:74)
        at org.python.core.Py.newString(Py.java:647)
        at org.python.core.PyJavaType.init(PyJavaType.java:585)
        at org.python.core.PyType.createType(PyType.java:1523)
        at org.python.core.PyType.addFromClass(PyType.java:1462)
        at org.python.core.PyType.fromClass(PyType.java:1551)
        at org.python.core.PyObject.<init>(PyObject.java:93)
        at org.python.core.PySingleton.<init>(PySingleton.java:9)
        at org.python.core.PyNotImplemented.<init>(PyNotImplemented.java:10)
        at org.python.core.Py.<clinit>(Py.java:94)
```

**Comment by wetneb** [comment]
@srht do you have any non-ASCII characters in your username on this computer?

**Comment by jackyq2015** [comment]
Might relate to this issue:
http://bugs.jython.org/issue2632

**Comment by thadguidry** [comment]
@jackyq2015 I'd say that's exactly the issue the OP has here.  Hmm, Jython team says this fix is going into Jython 2.7.2 and I would say we probably could just build our own Jython 2.7.2-SNAPSHOT from latest Jython and incorporate the last 6 months of their fixes from Stefan and Jeff https://hg.python.org/jython/shortlog/default

**Comment by jackyq2015** [comment]
@thadguidry Build a snapshot is an option but may introduce other bugs. I would prefer to wait until it get stabled and release an official version unless this particular bug is a show-stopper.

**Comment by thadguidry** [comment]
@jackyq2015 I reviewed the last 5 months of commits.  There's only fixes, and nothing I'd be worried about.  There's an alpha version floating on the mailing list. But I'd say even a beta version would be fine for us if your that worried, but I'm not as I stated.  Problem historically has been that Jython are slow on releases...but fast on bug fixes in master as they are found, only pushing new features once Python gets them.  But up to the community voices on this one.
